components/google_appsheet/README.md

```diff
@@ -6,3 +6,5 @@ computers. With Appsheet, you can easily create apps that allow you to track
 and manage your data, create reports and visualizations, and even share your
 data with other users. Appsheet is also easy to use, with a simple interface
 that makes it easy to get started.
+
+Pipedream's use and transfer of information received from Google APIs to any other app will adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Comment:
```suggestion
Pipedream's use and transfer of information received from Google APIs to any other app adhere to the [Google API Services User Data Policy](https://developers.google.com/terms/api-services-user-data-policy#additional_requirements_for_specific_api_scopes), including the Limited Use requirements.
```

Reply:
Changed "will adhere" to "adhere" to follow Google's Developer docs style guide (use present tense).